test/helpers/kubectl.go

```diff
@@ -286,42 +304,45 @@ func (kub *Kubectl) CiliumEndpointPolicyVersion(pod string) map[string]int64 {
 	return result
 }
 
-//CiliumExec runs cmd in the specified Cilium pod
+// CiliumExec runs cmd in the specified Cilium pod.
 func (kub *Kubectl) CiliumExec(pod string, cmd string) *CmdRes {
-	cmd = fmt.Sprintf("kubectl exec -n kube-system %s -- %s", pod, cmd)
+	cmd = fmt.Sprintf("%s exec -n kube-system %s -- %s", kubectl, pod, cmd)
 	return kub.Node.Exec(cmd)
 }
 
-//CiliumNodesWait When a new cilium DS is set, the status can be ok but tunnels
-//are not in place yet. Checking the `kubectl get nodes` we can know if cilium
-//added the node correctly. In case of timeout will return an error
+// CiliumNodesWait waits up until the specified timeout has elapsed
+// until all nodes in the Kubernetes cluster are annotated  with Cilium
```

Comment:
I found the above run-on combination of descriptions a little confusing. The function doesn't take a timeout, so maybe describing it as 'specified' is a little unclear.  Here's an attempt, feel free to take it or parts of it or just leave it alone if you don't think it helps:

```
CiliumNodesWait waits until all nodes in the Kubernetes cluster are annotated with Cilium 
annotations.  Its runtime is bounded by a maximum of `HelperTimeout`. When a node is 
annotated with Cilium annotations, it indicates...
```